src/Command/GenerateObjectAclCommand.php

```diff
@@ -94,7 +97,7 @@ public function configure()
             ->addOption('object_owner', null, InputOption::VALUE_OPTIONAL, 'If set, the task will set the object owner for each admin.')
             // NEXT_MAJOR: Remove "user_entity" option.
             ->addOption('user_entity', null, InputOption::VALUE_OPTIONAL, '<error>DEPRECATED</error> Use <comment>user_model</comment> option instead.')
-            ->addOption('user_model', null, InputOption::VALUE_OPTIONAL, 'Shortcut notation like <comment>AcmeDemoBundle:User</comment>. If not set, it will be asked the first time an object owner is set.')
+            ->addOption('user_model', null, InputOption::VALUE_OPTIONAL, 'Fully qualified class name <comment>App\Entity\User</comment>. If not set, it will be asked the first time an object owner is set.')
```

Comment:
IMO, we should avoid terms ("Entity", "Document") that are tied to a specific model manager.

Reply:
Model then ?

Reply:
I think yes. We haven't many choices.